rust/chains/hyperlane-cosmos/src/multisig_ism.rs

```diff
@@ -78,7 +77,7 @@ impl MultisigIsm for CosmosMultisigIsm {
         let validators: ChainResult<Vec<H256>> = response
             .validators
             .iter()
-            .map(|v| H160::from_str(v).map(h160_to_h256).map_err(Into::into))
+            .map(|v| H256::from_str(v).map_err(Into::into))
```

Comment:
I believe H256::from_str panics if the input length isn't 32 bytes, so we'll probably need to keep doing H160::from_str(v) and then .into() to H256

Reply:
great point